### ISSUE TITLE
[Feature Request] Consider using manylinux instead of musl to support multiple targets

### ISSUE THREAD

**Issue by WorksButNotTested** [luke-goddard/enumy#29]
The manylinux project is used as a basis for native python PIP modules to build binaries which run on many linux distributions. This may provide a simpler/more compatible alternative to musl?

https://github.com/pypa/manylinux/blob/master/docker/Dockerfile-x86_64

**Comment by luke-goddard** [comment]
Yeah, that seems like a great alternative for X86 and X64 releases, but I want to eventually support compiling to ARM. I think I will have to use musl for ARM compilation. Not sure, not done much research yet. I definitely like the idea of getting rid of musl though.
